.eslintrc

```diff
@@ -312,6 +312,14 @@
       "rules": {
         "max-len": "off"
       }
+    },
+    {
+      "files": ["scripts/**/*"],
+      "rules": {
+        "import/no-extraneous-dependencies": "off",
+        "no-console": "off",
+        "require-jsdoc":"off"
+      }
```

Comment:
Why you added these three rules for scripts folder?

Reply:
**import/no-extraneous-dependencies:** The Scripts are using some of the dependencies which are listed in DevDependencies in package.json file. Since the scripts are ran in the server, it does not matter where the dependencies are listed in package.json file. 

This error was occuring in the `scripts/compose.ts` file

![image](https://github.com/user-attachments/assets/375492ec-ae23-4ca7-abd8-f11cbb075d3a)

 

**no-console:** We have several console.log statements in the scripts, which are necessary for build scripts. So I have removed the eslint warning since it is evident for the build scripts to have console statements in them (view [your old comment](https://github.com/asyncapi/website/pull/3423#discussion_r1859134534)) 


**require-jsdoc:** We are migrating the files to typescript, I have removed the jsdoc rule because it would be irrelevant to keep jsdoc in typescript files.